src/types/response/rebate.ts

```diff
@@ -19,6 +19,13 @@ export interface AgencyCommissionHistoryRecord {
   source: string;
 }
 
+export interface PartnerSubordinateListRecord {
```

Comment:
```suggestion
export interface PartnerSubordinate {
```

I would rename this to what it is